samples/TupleToJSONConvert/Makefile

```diff
@@ -1,20 +1,20 @@
-# Copyright (C) 2011,2013,2014, International Business Machines Corporation. 
+# Copyright (C) 2011,2013,2016, International Business Machines Corporation. 
 # All Rights Reserved.
 
-.PHONY: all standalone clean 
+.PHONY: all build clean 
 
-STREAMS_MSG_TOOLKIT ?=../../com.ibm.streamsx.json
-SPLC_FLAGS ?= -a -t $(STREAMS_MSG_TOOLKIT)
+STREAMS_JSON_TOOLKIT ?=../../com.ibm.streamsx.json:$(STREAMS_INSTALL)/toolkits/com.ibm.streamsx.json
```

Comment:
I misread it, I thought that it now depended on the toolkit being in `$STREAMS_INSTALL`.

However, removing standalone mode is somewhat of an issue, as it seems to really designed to work in standalone, since it prints to standard out.